src/applications/debt-letters/tests/e2e/debt-letters.cypress.spec.js

```diff
@@ -10,7 +10,7 @@ describe('Debt Letters', () => {
     cy.axeCheck();
   });
 
-  it('C1033 displays the current debts section and navigates to debt details', () => {
+  it.skip('C1033 displays the current debts section and navigates to debt details', () => {
```

Comment:
Skips are because e2e run against a prod build and this PR sets vagovprod to false